src/applications/financial-status-report/sass/financial-status-report.scss

```diff
@@ -3,7 +3,6 @@
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-form-process";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-modal";
 @import "~@department-of-veterans-affairs/formation/sass/modules/m-omb-info";
-@import "~@department-of-veterans-affairs/formation/sass/shared-variables";
```

Comment:
Why did we remove shared-variables here?

Reply:
It's a duplicate reference, that one is already defined on line 1
![Screenshot 2023-05-05 at 8 40 36 AM](https://user-images.githubusercontent.com/3916436/236504401-253aa05f-2ec2-49d6-bfae-6abf4502db63.png)